internal/client/machine_pool.go

```diff
@@ -22,6 +22,7 @@ type MachinePool struct {
 	PrimaryDiskDevice          string                 `json:"primary_disk_device"`
 	ReuseExistingRootPartition bool                   `json:"reuse_existing_root_partition"`
 	Status                     string                 `json:"status"`
+	NetworkProfileId           *int64                 `json:"network_profile_id"`
```

Comment:
https://go.dev/wiki/CodeReviewComments#initialisms

```suggestion
	NetworkProfileID           *int64                 `json:"network_profile_id"`
```